modules/S28_python_run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# EMBA - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2020-2024 Siemens Energy AG
+#
+# EMBA comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# EMBA is licensed under GPLv3
+#
+# Author(s): Michael Messner, Thomas Gingele
+#
+# Description:  This is an experimental EMBA module. It is designed to run user-defined python
+#               scripts during the analysis.
+#
+# shellcheck disable=SC2004
+
+S28_python_run() {
+    module_log_init "${FUNCNAME[0]}"
+    module_title "Python Runner"
+    pre_module_reporter "${FUNCNAME[0]}"
+
+    local lSCRIPT_DIR="./modules/S28_python_run"
+    local lPYTHON_SCRIPT_COUNT=${#PYTHON_SCRIPTS[@]}
+    local lPYTHON_BIN=""
+    local lRESULTS=()
+    local lCOUNT_FINDINGS=0
+
+    lPYTHON_BIN="$( find . -name python3 | head -n 1 )"
```

Comment:
We need to check this in the dependency checker. I also think this find is not needed as the installer should have already installed python3 in path. So, at the end we can do something like this here

https://github.com/e-m-b-a/emba/blob/d829e18ceba639e6d990d9bb7dadadb0ae462117/helpers/helpers_emba_dependency_check.sh#L709